.github/workflows/test.yaml

```diff
@@ -89,8 +89,9 @@ jobs:
         uses: matter-labs/era-compiler-ci/.github/actions/build-llvm@v1
         with:
           target-env: ${{ contains(matrix.target, 'musl') && 'musl' || 'gnu' }}
-          enable-assertions: 'false'
           enable-coverage: ${{ matrix.enable-coverage || 'false' }}

```

Comment:
@antonbaliasnikov why this line is missing in the corresponding zksolc workflow?

Reply:
good catch. I removed this line for `zksolc`, but kept for `zkvyper`.

As we use `codecov` for coverage, it automatically matches the codebase with the coverage report.
So, it doesn't make sense here to build LLVM with coverage, only Rust part suffices.

The LLVM part is measured in full by the compiler tester in the LLVM repo.

So, we need to remove `enable-coverage` from LLVM build step at all.

Reply:
@antonbaliasnikov done! Pls resolve & approve.